### ISSUE TITLE
Contractions being marked as typos

### ISSUE THREAD

**Issue by winhamwr** [PolicyStat/ckeditor-spell-check-plugin-js-dev-edge#30]
It seems like most (all?) contractions are being marked as typos. All of these examples were in my document on initial load, so it's not any kind of interaction issue.
### doesn't

``` html
<li><strong>Wait a month</strong>: It probably <span class="nanospell-typo">doesn</span>'t make sense for us to wait a month to prioritize that typo fix as part of an <span class="nanospell-typo">OKR</span> cycle. We want to be more responsive than that.</li>
```
### shouldn't

``` html
<p>For all other product enhancements (which includes bug fixes), the best way to <strong>increase the visibility is to increase the number of votes the idea has in UserVoice</strong>. The best way to do this is to record votes for any customers you speak with who express interest in your idea. You <span class="nanospell-typo">shouldn</span>'t be afraid to bring it up with them while talking about something else.&nbsp;You can also campaign internally to get other team members to vote on it.</p>
```
### aren't

``` html
<p>Votes <span class="nanospell-typo">aren</span>'t a perfect system, but they're a concrete way of quantifying the scope of an idea's impact. It does have some distinct advantages over fuzzier systems.</p>
```

**Comment by caffodian** [comment]
This seems way easier to fix than the other one, which has magically become unreproducible for me since I last tested it last afternoon.  :/

**Comment by caffodian** [comment]
In this case, quotes are incorrectly separating words, which makes sense sometimes, but not always

**Comment by caffodian** [comment]
quick fix is probably just to make ' not a separator.

**Comment by caffodian** [comment]
and of course, we have multiple regexes in different places.  argh

**Comment by caffodian** [comment]
Removing single quote shoud be relatively harmless, the only thing it would break is if someone wrote a document and used single quotes without spaces around them everywhere.  which would be dumb.

**Comment by caffodian** [comment]
Need to do more testing with this change with actual policies to see if anything breaks.  I don't think it breaks anything in spellcheck itself.

**Comment by caffodian** [comment]
Fixed in #31
